src/services/pcn-dynmon/src/Dynmon.h

```diff
@@ -6,13 +6,25 @@
 #include "models/DataplaneConfig.h"
 #include "models/Metric.h"
 #include "models/Metrics.h"
+#include "swap/SwapCompiler.h"
```

Comment:
I would not call it a "compiler". A compiler is something else and this may confuse people. 
Let's call it `SrcCodeRewriter` or just `Rewriter`, it gives a better idea of what this component does.

Reply:
Agree, I'm going to change it asap :)

Reply:
Solved in 523ff8b